### PR TITLE
Expanded Rhino Captain arsenal

### PR DESCRIPTION
**Pull request by KingSuperBOB**
## About The Pull Request
Two new captain rhinos which are randomized roundstart, rhino captain mechs now need specifically rhino captain access, slight light rhino buff, rhino belt for error repairs, removal of rhino acid armor.

## Why It's Good For The Game
Further round variety, light rhinos are now slightly viable, low ranking pilots dont steal captain mechs, rhinos can now fix the fire burning them alive and xenos can now actually kill rhinos.

## Changelog
:cl:
add: Melee Captain Rhino, Heavy captain rhino, Rhino belts
tweak: Rhino Captain is a actual thing instead of a var edit
balance: all rhinos have acid armor 0, Durand gone from VII/VII/VII/I to V/V/V/I
mapping: removed rhino pilot gear from the map, removed var edited captains rhino, added randomize spawner for captain rhino
/:cl:

### CODE REVIEW

**Comment by KingSuperBOB** [comment]
Will need to fix linters

File: ModularTegustation/tegu_items/rcorp/itemspawner.dm [hunk 1]
```diff
@@ -92,6 +92,26 @@
...
+	icon = 'icons/effects/landmarks_static.dmi'
+	icon_state = "x4"
+	var/list/possible_mecha = list(
+		/obj/vehicle/sealed/mecha/combat/rhino/captain,
+		/obj/vehicle/sealed/mecha/combat/rhinomelee/captain,
+		/obj/vehicle/sealed/mecha/combat/durand,
+	)
+
+/obj/effect/landmark/rhinocaptainspawner/Initialize()
+	..()
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/rcorp/itemspawner.dm:108]: Do not leave initialize without a return hint, always do eighter `. = ..()`, `return ..()` or sometimes `return INITIALIZE_HINT_[insert case here]`
```suggestion
	. = ..()
```

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
The durand shield is infinite and makes it invincible lol

**Comment by KingSuperBOB** [comment]
Durand shield given 15 second cooldown

**Review by Kitsunemitsu — APPROVED** [review]
Fixed ya spelling mistake G.

This looks good, if not great.
Rcorp is in for a buff and this is a great way to do it!

**Pull request merged**